packages/messenger/src/Messenger.test.ts

```diff
@@ -27,6 +27,24 @@ describe('Messenger', () => {
       expect(count).toBe(1);
     });
 
+    it('allows registering and calling an action handler for a different namespace using DISABLE_NAMESPACE', () => {
+      type CountAction = {
+        type: 'Fixture:count';
+        handler: (increment: number) => void;
+      };
+      const messenger = new Messenger<string, CountAction, never>({
+        namespace: DISABLE_NAMESPACE,
+      });
```

Comment:
Continuing with another comment, what do you think about adding a type alias for `string`, `AnyNamespace`? Then combined with the renaming of the constant, this would look like:

``` suggestion
      const messenger = new Messenger<AnyNamespace, CountAction, never>({
        namespace: ANY_NAMESPACE,
      });
```

Alternatively:

``` suggestion
      const messenger = new Messenger<AllNamespaces, CountAction, never>({
        namespace: ALL_NAMESPACES,
      });
```

Reply:
I've added a `MockAnyNamespace` type here: 6e633510c68ad69ea66f7e70120f6a7520e372e9